MantidQt/CustomInterfaces/src/Indirect/Elwin.cpp

```diff
@@ -496,6 +493,54 @@ void Elwin::updateRS(QtProperty *prop, double val) {
     backgroundRangeSelector->setMaximum(val);
 }
 
+/**
+ * Handles mantid plotting
+ */
+void Elwin::plotClicked() {
+
+  auto outputWs = m_elwinAlg->getPropertyValue("OutputInQ");
+  checkADSForPlotSaveWorkspace(outputWs, true);
+  plotSpectrum(QString::fromStdString(outputWs));
+
+  auto outputWsSquared = m_elwinAlg->getPropertyValue("OutputInQSquared");
+  checkADSForPlotSaveWorkspace(outputWsSquared, true);
+  plotSpectrum(QString::fromStdString(outputWsSquared));
+
+  auto elfWs = m_elwinAlg->getPropertyValue("OutputELF");
+  checkADSForPlotSaveWorkspace(elfWs, true);
+  plotSpectrum(QString::fromStdString(elfWs), 0, 9);
+
+  if (m_blnManager->value(m_properties["Normalise"])) {
+    auto eltWs = m_elwinAlg->getPropertyValue("OutputELT");
+    checkADSForPlotSaveWorkspace(eltWs, true);
```

Comment:
When setting `Normalise to Lowest Temp` to `True`, I get an error:

```Error while plotting: The workspace "irs26176_graphite002_elwin_elt" could not be found```

as that workspace is not in the ADS.